assets/css/dup_v2.scss

```
@@ -28,6 +28,8 @@
 
 @import "v2/multi_screen_page";
 @import "v2/dup/simulation";
+@import "v2/dup/debug.scss";
+@import "v2/dup/empty_state.scss";
```

Drop file type for consistency.

```suggestion
@import "v2/dup/debug";
@import "v2/dup/empty_state";
```